src/data/fillChainDetails.py

```diff
@@ -67,7 +67,7 @@ def get_chain_ids(contracts):
     return [
         chain_id
         for chain_id in chain_ids
-        if any(contracts[chain_id].get(f"v{i}") for i in range(3, 10))
+        # if any(contracts[chain_id].get(f"v{i}") for i in range(3, 10))
```

Comment:
? Should this line be deleted?

Reply:
yeah, legacy, makes no sense. if there's a contract deployed on a chain, we should get the chaindetails. Else we should just remove the contract from contracts.json instead